CHANGELOG.md

```diff
@@ -106,6 +106,8 @@ v0.38.0-rc.0 (2023-11-16)
 
 - Updated windows exporter to use prometheus-community/windows_exporter commit 1836cd1. (@mattdurham)
 
+- Added support to convert resource attributes to Prometheus-compatible datapoint attributes in `otelcol.exporter.prometheus`. (@hainenber)
```

Comment:
```suggestion
- Added support to convert resource attributes to Prometheus labels in `otelcol.exporter.prometheus`. (@hainenber)
```
I think "attribute" and "datapoint" are OTel terms, so probably it's good enough just to say its a Prometheus label.